src/vmspawn/vmspawn-creds.c

```diff
@@ -0,0 +1,25 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+
+#include "alloc-util.h"
+#include "macro.h"
+#include "memory-util.h"
+#include "vmspawn-creds.h"
+
+static void credential_free(Credential *cred) {
```

Comment:
Hmm, this is a copy of `nspawn-creds.[ch]`. Please move that to e.g. `shared/machine-credential.[ch]` rather than duplicating it.